build/assets/custom-dictionary.txt

```diff
@@ -1,49 +1,672 @@
-personal_ws-1.1 en 1315 utf-8
```

Comment:
This header line needs to appear at the top.  The 1315 is an approximate word count for the rest of the dictionary.